src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -766,6 +796,37 @@ export const gravityStitchingEnvironment = (
           },
         },
       },
+      Query: {
+        curatedMarketingCollections: {
+          fragment: gql`
+            ...on Viewer {
+              __typename
+            }
+          `,
+          resolve: async (_parent, _args, context, info) => {
```

Comment:
**suggestion**: I think that we want to pass these args to the delegated query below so that `size` works as expected (see args comment above).

Reply:
I don't know how it works, but size limiting was working even without explicitly passing `args` to `delegateToSchema`. I've added `...args` though to make is explicit